### PR TITLE
Update recipe based on template

### PR DESCRIPTION
**Pull request by aosthof**

### CODE REVIEW

**Review by rsimai — APPROVED** [review]
looks very good. I just wonder who owns the Schäufele trademark :-)

File: recipes/main/schaeufele.adoc [hunk 1]
```diff
@@ -19,23 +19,23 @@ the pan of a shovel and comes from the shape of the bone (pork shoulder).
...
-| 1/2 | 1/2 | Celery root
-| 2 | 2 | Garlic cloves
-| | | Parsley (twigs)
-| 4 cups | 1 l | Broth
-| 7 fl oz | 200 ml | Beer (e.g. lager, brown ale, not hoppy)
+| US | Metric | Ingredient | Zutat
+
+| 3-4 lb | 1,5-2 kg | Pork shoulder with bone and rind | Schäufele™
+| | | Salt | Salz
+| | | Black pepper (freshly grounded) | Schwarzer Pfeffer (frisch gemahlen)
```
Comment (dcermak) [inline recipes/main/schaeufele.adoc]:
```suggestion
| | | Black pepper (freshly ground) | Schwarzer Pfeffer (frisch gemahlen)
```

File: recipes/main/schaeufele.adoc [hunk 2]
```diff
@@ -19,23 +19,23 @@ the pan of a shovel and comes from the shape of the bone (pork shoulder).
...
+| | | Caraway (grounded) | Kümmel (gemahlen)
```
Comment (dcermak) [inline recipes/main/schaeufele.adoc]:
```suggestion
| | | Caraway seeds (grounded) | Kümmel (gemahlen)
```

Comment (aosthof) [reply recipes/main/schaeufele.adoc]: Thanks for spotting these, fixed in latest commit.

**Pull request merged**